src/lib/lib.jl

```diff
@@ -46,10 +46,14 @@ end
 # backpropagated as values; these should be mutually exclusive options.
```

Comment:
Can this TODO be removed now?

Reply:
Huh, guess I already thought of this issue at some point :D Yeah I think this is the fix though.